package.json

```diff
@@ -19,7 +19,8 @@
   "dependencies": {
     "accepts": "^1.3.3",
     "debug": "^2.2.0",
-    "ejs": "^2.4.2",
+    "ejs": "^2.5.5",
```

Comment:
AFAICT, the latest `ejs` version is `2.5.7`. Please use that version instead of `2.5.5`.